src/components/editUserInfoModal/index.jsx

```diff
@@ -0,0 +1,153 @@
+        return <>Loading...</>;
+    }
+    return (
+        <Modal title="Edit user info"
+            visible={true}
+            onCancel={() => close()}
+            cancelButtonProps={{ style: { display: 'none' } }}
+            okButtonProps={{ style: { display: 'none' } }}
+            footer={null}
+        >
+            <Form
+                labelCol={{
+                    span: 5
+                }}
+                wrapperCol={{
+                    span: 32
+                }}
+                onFinish={onFinish}
+                onFinishFailed={onFinishFailed}
+                scrollToFirstError
+            >
+                <Form.Item name="name"
+                    label={"Name: "}
+                    initialValue={userData?.name}
+                    rules={[
+                        InputRules.latinLetters(
+                            inputValidationErrorMessages.NOT_VALID_NAME
+                        ),
+                        InputRules.lengthRange(
+                            2,
+                            50,
+                            userErrorMessages.USER_NAME_LENGTH_RANGE
+                        ),
+                        InputRules.required(
+                            inputValidationErrorMessages.EMPTY_USER_NAME
+                        )
+                    ]}
+                >
+                    <Input />
+                </Form.Item>
+
+                <Form.Item name="surname"
+                    label={"Surname: "}
+                    initialValue={userData?.surname}
+                    rules={[
+                        InputRules.latinLetters(
+                            inputValidationErrorMessages.NOT_VALID_SURNAME
+                        ),
+                        InputRules.lengthRange(
+                            2,
+                            50,
+                            userErrorMessages.USER_NAME_LENGTH_RANGE
+                        ),
+                        InputRules.required(
+                            inputValidationErrorMessages.EMPTY_USER_SURNAME
+                        )
+                    ]}
+                >
+                    <Input />
+                </Form.Item>
+
+                <Form.Item name="email"
+                    label={"Email: "}
+                    initialValue={userData?.email}
+                    rules={[
+                        InputRules.specificType(
+                            "email",
+                            inputValidationErrorMessages.NOT_VALID_EMAIL
+                        ),
+                        InputRules.required(
+                            inputValidationErrorMessages.EMPTY_USER_EMAIL
+                        )
+                    ]}
+                >
+                    <Input />
+                </Form.Item>
+
+                <Form.Item>
+                    <Button
+                        block
+                        htmlType={"submit"}
+                        type="primary"
```

Comment:
I think, it`s better to add className="submitButton" for Button to make it in style of project.